master/static/migrations/20221213084307_searcher-metric-in-explist.tx.up.sql

```diff
@@ -0,0 +1,10 @@
+
+alter table trials add column if not exists searcher_metric_value float8 default null;
+
+update trials t set
+    searcher_metric_value = (
+      select (metrics->'validation_metrics'->>(e.config->'searcher'->>'metric'))::float8
+      from validations where id = t.best_validation_id
+    )
+from trials
+join experiments e on e.id = trials.experiment_id
```

Comment:
If we _did_ want to do an index though, it would be best to do a partial index `WHERE trial_state ='COMPLETED'`.

Since, in general, we expect in-progress trials to get better and better, updating an index for the 'best validation,' would be costly.